lib/simple_form/components/input_groups.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+module SimpleForm
+  module Components
+    module InputGroups
+      module Prepends
+        def prepend(wrapper_options = nil)
```

Comment:
Could we add tests to it?

Reply:
Would you help me with the test setup? There is no components test I could cheat on.

Reply:
To use this feature we need to have a wrapper? How I build a input using it?